src/main/kotlin/net/atos/zac/zaak/ZaakService.kt

```diff
@@ -66,7 +66,7 @@ class ZaakService @Inject constructor(
                         bepaalRolMedewerker(it, zaak),
                         explanation
                     )
-                }
+                } ?: zrcClientService.deleteRol(zaak, BetrokkeneType.MEDEWERKER, explanation)
```

Comment:
Checking: are you sure it should be this betrokkene type?

Reply:
I think so, it's the same role we use in the releasement function